src/panel/poi/PoiBlockContainer.jsx

```diff
@@ -43,6 +43,17 @@ export default class PoiBlockContainer extends React.Component {
       {displayCovidInfo &&
         <CovidBlock block={covidBlock} countryCode={this.props.poi.address.country_code} />}
       <Divider />
+      {
+        (
+          (this.props.poi.address && this.props.poi.subClassName !== 'latlon')
+          || websiteBlock
```

Comment:
I've been wondering for some time if there is a more elegant way to write this, and generally the block by block stuff. Sure there is, but it would require some re-org of the files first, and managing some exceptions, so :+1: for being pragmatic :smile: